src/Components/Timer.tsx

```diff
@@ -34,28 +13,38 @@ const Timer = () => {
   const gamePaused = useGamePausedDocState();
   const weaponsStolen = useWeaponsStolenDocState();
   const time = new Date();
-  const [soundOn, setSoundOn] = useState<boolean>(true)
+  const { gameAudio, loadAndPlayEscapeSoundtrack, playWarningSound, musicOn, soundOn } = useAudio();
 
   useEffect(() => {
+
     if (weaponsStolen.length === 4) {
       gameAudio.pause();
-      playWarning()
 
-      const warningTimer = setTimeout(async () => {
-        gameAudio = loadEscapeSoundtrack()
-        gameAudio.play();
-      }, 5100);
+      if (!musicOn) { 
+        loadAndPlayEscapeSoundtrack(); // load but don't play
```

Comment:
seems to me like loadAndPlayEscapeSoundtrack()
will not play the soundtrack. dont see .play() in the function.
unless you tested and it works.

I reckon we split it up, have a function to loadEscapeSoundtrack
and another method to playSoundtrack() (playSoundtrack more generic, that plays whichever soundtrack is loaded into gameAudio)

Reply:
nah it works cause the game audio plays when its changed and its changed in load escape soundtrack
Below is in `useSounds.tsx`
```
  useEffect(() => {
    if (musicOn) gameAudio.play();
    else gameAudio.pause();
  }, [musicOn, gameAudio])

  const loadAndPlayEscapeSoundtrack = () => {
    const escapeSoundtrack = new Audio(escapeSound);
    escapeSoundtrack.loop = true;
    setGameAudio(escapeSoundtrack);
  };